boards/snps/nsim/arc_v/Kconfig.defconfig

```diff
@@ -6,7 +6,11 @@ if BOARD_NSIM_ARC_V_RMX100
 config SYS_CLOCK_TICKS_PER_SEC
 	default 1000
 
-config SYS_CLOCK_HW_CYCLES_PER_SEC
-	default 5000000
-
 endif # BOARD_NSIM_ARC_V_RMX100
+
+if BOARD_NSIM_ARC_V_RHX100
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 10000
```

Comment:
10000 is a very high number for this. 100 or 1000 is the normal value. With tickless kernel the higher value is probably ok, but without that's a lot of time ISR processing...